packages/server/src/bin.ts

```diff
@@ -28,12 +29,20 @@ const run = async () => {
     blockstore = new MemoryBlockstore()
   }
 
+  let createDbTables = false;
+
   if (cfg.databaseLocation) {
+    createDbTables = !fs.statSync(cfg.databaseLocation, {throwIfNoEntry: false});
```

Comment:
Just a headsup that the linter will ask it to look more like this:
```suggestion
    createDbTables = !fs.statSync(cfg.databaseLocation, { throwIfNoEntry: false });
```

With #256 it should start getting flagged in CI, though.